jobs/JGFS_TC_TRACK_AVNO

```diff
@@ -35,17 +34,18 @@ export pgmerr=errfile
 # SENDCOM  - Copy Files From TMPDIR to $COMOUT
 # SENDDBN  - Issue DBNet Client Calls
 ####################################
-export SENDECF=${SENDECF:-YES}
+export SENDECF=${SENDECF:-NO}
 export SENDCOM=${SENDCOM:-YES}
-export SENDDBN=${SENDDBN:-YES}
+export SENDDBN=${SENDDBN:-NO}
```

Comment:
Set defaults for operational use (most likely "YES", double check) and send in overrides as needed for testing.